deckhouse-controller/pkg/apis/deckhouse.io/v1alpha1/deckhouse_release.go

```diff
@@ -25,15 +25,15 @@ import (
 )
 
 var (
-	DeckhouseReleaseGVR = schema.GroupVersionResource{
-		Group:    SchemeGroupVersion.Group,
-		Version:  SchemeGroupVersion.Version,
-		Resource: "deckhousereleases",
-	}
 	DeckhouseReleaseGVK = schema.GroupVersionKind{
 		Group:   SchemeGroupVersion.Group,
 		Version: SchemeGroupVersion.Version,
-		Kind:    "DeckhouseRelease",
+		Kind:    DeckhouseReleaseKind,
+	}
+	DeckhouseReleaseGVR = schema.GroupVersionResource{
+		Group:    SchemeGroupVersion.Group,
+		Version:  SchemeGroupVersion.Version,
+		Resource: DeckhouseReleasesKind,
```

Comment:
It's not a kind, it's a resource. It must be lowercase and plural form
```suggestion
		Resource: "deckhousereleases",
```

Reply:
Fixed. Actually this structure is not used